### PR TITLE
DT-57-fix-handle-token-login

### PR DESCRIPTION
**Pull request by lamxuanthinh**
I have completed set up type when call api that handle sign-up authentication . Please help me check now , @Giang-st1-00 @DuyTinh2003  Thanks !

### CODE REVIEW

File: Views/SignIn/index.tsx [hunk 1]
```diff
@@ -105,7 +109,7 @@ export default function Login() {
                                     type="submit"
                                     className="w-full font-bold text-[20px] bg-primary text-white bg-black hover:bg-[#3d3d3d] px-5 py-3 rounded-xl"
                                 >
-                                    Login
+                                    Sign in
```
Comment (Giang-st1-00) [inline Views/SignIn/index.tsx:110]: Sign in ? I think it should be named login

File: Views/SignIn/index.tsx [hunk 2]
```diff
@@ -119,8 +123,10 @@ export default function Login() {
                                 type="submit"
                                 className="border-[#52525233] border-2 rounded-xl w-full bg-primary bg-white px-5 py-3 flex flex-row justify-center items-center"
                             >
-                                <FcGoogle className="text-2xl" />
-                                <p className="font-bold text-[20px] px-0 sm:px-5 ml-2 dark:text-black">Login with Google</p>
+                                <FcGoogle className="text-2xl"/>
+                                <p className="font-bold text-[20px] px-0 sm:px-5 ml-2 dark:text-black">
+                                    Sign in with Google
```
Comment (Giang-st1-00) [inline Views/SignIn/index.tsx:126]: same as above

File: Views/SignIn/index.tsx [hunk 3]
```diff
@@ -1,42 +1,43 @@
...
-    const { setIsLoading } = useAppContext();
+    const {setIsLoading} = useAppContext();
     const [errorEmail, setErrorEmail] = useState<string>("");
     const [errorPassword, setErrorPassword] = useState<string>("");
 
     const {
         register,
         handleSubmit,
-        formState: { errors },
+        formState: {errors},
```
Comment (Giang-st1-00) [inline Views/SignIn/index.tsx]: Why remove the spaces? I think auto format is correct.

File: Views/SignIn/index.tsx [hunk 4]
```diff
@@ -55,7 +56,8 @@ export default function Login() {
 
     return (
         <div className="h-screen w-screen bg-white md:bg-[#c3c3c3f5] flex justify-center items-center">
-            <div className="max-w-[600px] lg:max-w-none w-full lg:w-[1056px] h-[700px] rounded-2xl bg-[#fff] flex justify-between p-10 md:p-5 md:pl-10 overflow-hidden">
+            <div
+                className="max-w-[600px] lg:max-w-none w-full lg:w-[1056px] h-[700px] rounded-2xl bg-[#fff] flex justify-between p-10 md:p-5 md:pl-10 overflow-hidden">
```
Comment (Giang-st1-00) [inline Views/SignIn/index.tsx]: what? Your code format is strange

**Comment by Duy-Tinh-Dev** [comment]
Oke

File: Views/SignIn/index.tsx (hunk 1 above)
Comment (lamxuanthinh) [reply Views/SignIn/index.tsx:110]: Screenshot 2023-09-23 at 22 44 46
I think `sign-up` good than `login` @Giang-st1-00

File: Views/SignIn/index.tsx (hunk 3 above)
Comment (lamxuanthinh) [reply Views/SignIn/index.tsx]: I think we should install 'pretty,' which would help resolve this problem

File: Views/SignIn/index.tsx (hunk 4 above)
Comment (lamxuanthinh) [reply Views/SignIn/index.tsx]: Because of , I used Intellij 👍

Comment (lamxuanthinh) [reply Views/SignIn/index.tsx]: @Giang-st1-00 I have fixed

File: Views/SignIn/index.tsx (hunk 3 above)
Comment (lamxuanthinh) [reply Views/SignIn/index.tsx]: @Giang-st1-00 I have fixed

**Pull request merged**